.travis.yml

```diff
@@ -2,5 +2,15 @@ language: node_js
 node_js:
   - 'stable'
   - 'lts/*'
-
-script: npm run lint && npm run test
+before_script:
+  - 'npm install grunt-cli -g'
+  - 'export DISPLAY=:99.0'
+  - 'sh -e /etc/init.d/xvfb start'
+  - 'sleep 3'
+script:
+  - 'npm run lint'
+  - 'npm run test'
+  - 'if [ ${TRAVIS_NODE_VERSION} = "stable" ]; then
```

Comment:
I make it that it runs only `stable` because there is no support of ES7 syntax in `lts/*`.